alica_engine/src/engine/RunningPlan.cpp

```diff
@@ -104,6 +105,17 @@ bool RunningPlan::isDeleteable() const
         return true; // shortcut for plans from planselector
     return isRetired() && (!isBehaviour() || !_ae->getBehaviourPool()->isBehaviourRunningInContext(*this));
 }
+
+void RunningPlan::preTick()
+{
+    if (isRetired()) {
+        return;
+    }
+    evalRuntimeCondition();
```

Comment:
This can throw std::exception, shouldn't we need to Abort or have a proper handling for exception?

Reply:
Hm? An exception during evaluation is caught in Condition::evaluate and should not reach this point.